internal/handlers/tigris/fetch.go

```diff
@@ -26,27 +26,21 @@ import (
 	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
 )
 
-// fetchParam represents options/parameters used by the fetch.
-type fetchParam struct {
-	db         string
-	collection string
-}
-
 // fetch fetches all documents from the given database and collection.
 //
 // TODO https://github.com/FerretDB/FerretDB/issues/372
-func (h *Handler) fetch(ctx context.Context, param fetchParam) ([]*types.Document, error) {
-	db := h.db.Driver.UseDatabase(param.db)
+func (h *Handler) fetch(ctx context.Context, param tigrisdb.FetchParam) ([]*types.Document, error) {
```

Comment:
While we change that, maybe we could move the whole `fetch` as well?

Reply:
Fixed